src/WarmUps/Cat.java

```diff
@@ -0,0 +1,17 @@
+package WarmUps;
+
+public class Cat {
+	String name;
+
+	public Cat(String name) {
+		this.name = name;
+	}
+	
+	public boolean equals(Object o) {
```

Comment:
you could condense this to a single line- return (((Cat)o).name.equals(this.name));